src/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(
+    #{$width-cell * $cells-in-row} + #{$gap-size * ($cells-in-row - 1)}
+  );
+  gap: $gap-size;
+
+  &__day {
+    height: $height-cell;
+    width: $width-cell;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    border: 1px solid black;
+    transition-duration: 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+    }
+  }
+
+  @for $month-length from 28 through 31 {
+    &--month-length-#{$month-length} {
+      .calendar__day {
```

Comment:
The selector `.calendar__day` should be written as `&__day` inside the nested block to maintain consistency with BEM methodology and the existing code structure. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)